app/models/duke-ds-project-transfer.js

```diff
@@ -5,6 +5,13 @@ export default DS.Model.extend({
   status: DS.attr('string'),
   statusComment: DS.attr('string'),
   toUsers: DS.hasMany('DukeDsUser',),
+  toUsersNames: Ember.computed('toUsers[].fullName', function() {
+    return this.get('toUsers').reduce(function(acc, value) {
+      if (acc) acc = acc.concat(", ");
+      acc = acc.concat(value.get('fullName'));
+      return acc
+    }, '')
+  }),
```

Comment:
If you use [mapBy](https://www.emberjs.com/api/ember/2.15/classes/Ember.Array/methods/mapBy?anchor=mapBy) and [join](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join), this becomes a pretty clear one-liner: 

```
toUsersNames: Ember.computed('toUsers[].fullName', function() {
  return this.get('toUsers').mapBy('fullName').join(', ');
}),
```

Reply:
And FYI a [computed.mapBy](https://www.emberjs.com/api/ember/2.15/namespaces/Ember.computed/methods/collect?anchor=mapBy) provides an array property that works for for display and filtering - it just doesn't render spaces after the commas:

```
toUsersNames: Ember.computed.mapBy('toUsers','fullName'),
```

One could argue the comma-space is presentation logic (not model) and should be in the component layer, but I don't know that I'd add take on the complexity of a custom component or [helper](https://guides.emberjs.com/v2.15.0/templates/writing-helpers/) for that.